data/tutorials/lg_08_error_handling.md

```diff
@@ -907,9 +914,10 @@ several other mainstream languages, OCaml provides several mechanisms to handle
 exceptional events, all with good runtime performance and code
 understandability. Using them properly requires some initial learning and
 practice. Later, it always requires some thinking, which is good since proper
-error management shouldn't ever be overlooked. No error handling is better
-than the others, and is should be matter of adequacy to the context rather than
-of taste. But opinionated OCaml code is also fine, so it's a balance.
+error management shouldn't ever be overlooked. No error handling mechanism is
+always better than the others, and choosing one to use should be matter of
+fitting the context rather than of taste. But opinionated OCaml code is also
+fine, so it's a balance.
```

Comment:
```suggestion
error management shouldn't ever be overlooked. No error handling mechanism is
always better than the others, and choosing one to use should be a matter of
fitting the context rather than that of taste. But opinionated OCaml code is also
fine, so it's a balance.
```